doc/DeveloperGuide.md

```diff
@@ -125,6 +125,21 @@ Use case ends.
 > 1c1. AddressBook shows an error message <br> 
   Use case ends
 
+#### Use case: Change Theme
+  **MSS**
```

Comment:
Please be careful in terms of the indentation and formatting (the indentation of the header and MSS did not follow the rest of the use cases)